maestro/tag-mapping.md

```
@@ -147,6 +176,13 @@ frontend/shared/src/commonMain/kotlin/dev/egograph/shared/ui/
 - tapOn: "api_url_input"
 - inputText: "https://api.egograph.dev"
 - tapOn: "save_settings_button"
+
+# ターミナル操作
+- tapOn: "session_item"
+- assertVisible: "terminal_back_button"
+- tapOn: "terminal_copy_button"
+- assertVisible: "terminal_copy_feedback"
+```
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)